system/COPY/etc/motd

```diff
@@ -0,0 +1,13 @@
+Welcome to the
+                   _ _
+  __ _ _ __  _ __ | (_) __ _ _ __   ___ ___
+ / _` | '_ \| '_ \| | |/ _` | '_ \ / __/ _ \
+| (_| | |_) | |_) | | | (_| | | | | (_|  __/
+ \__,_| .__/| .__/|_|_|\__,_|_| |_|\___\___|
+      |_|   |_|            _
+  ___ ___  _ __  ___  ___ | | ___
+ / __/ _ \| '_ \/ __|/ _ \| |/ _ \
+| (_| (_) | | | \__ \ (_) | |  __/
+ \___\___/|_| |_|___/\___/|_|\___|
```

Comment:
can we play <insert favorite 80s video game> theme music when we show that motd?